src/cryptonote_core/cryptonote_core.cpp

```diff
@@ -1077,6 +1077,9 @@ namespace cryptonote
                          std::chrono::milliseconds(500),
                          false,
                          m_pulse_thread_id);
+        m_lmq->add_timer([this]() {this->check_service_node_time();},
+                         std::chrono::seconds(30),
```

Comment:
We have the `std::literals` namespace in effect everywhere, which means you can use `30s` here for a duration literal (https://en.cppreference.com/w/cpp/chrono/duration#Literals).